Firmware/HomeButtonsArduino/src/app.h

```diff
@@ -47,11 +145,12 @@ class App : public Logger {
   void _publish_awake_mode_avlb();
   void _mqtt_callback(const char* topic, const char* payload);
   void _net_on_connect();
+  void _download_mdi_icons();
 
   enum class StateMachineState {
```

Comment:
Remove this enum `StateMachineState` + `sm_state_` (unused, write-only variables)